fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalApplyOperator.java

```diff
@@ -50,6 +50,14 @@ public class LogicalApplyOperator extends LogicalOperator {
 
     private final boolean needOutputRightChildColumns;
 
+    /**
+     * Record un-correlation subquery outer table column, use for push down apply node
+     * e.g.
+     * SQL: select * from t1 where t1.v1 > (....);
+     * OuterColumns: t1.v1
+     */
+    private final ColumnRefSet unCorrelationSubqueryOuterColumns;
```

Comment:
Why couldn't use output field?

Reply:
it's different, output means the result of subquery, this field means what columns will do predicate with subquery

Reply:
if so, unCorrelationSubqueryPredicateColumns is more better?